src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs/worker.py

```
@@ -97,193 +84,251 @@ def _compute_output_dir(self, title: str) -> Path:
+                )
+                item.status = status
+                item.error = _msg
+
+    def _save_files_stats(self, stats_data) -> None:
+        files_stats_path = self.output_dir / "files_stats.json"
+        try:
+            with open(files_stats_path, "w", encoding="utf-8") as f:
+                json.dump(stats_data, f, indent=4, ensure_ascii=False)
+
+        except (OSError, TypeError, ValueError) as e:
+            logger.error(f"Error saving json: {e}, path: {str(files_stats_path)}")
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)